CMakeLists.txt

```diff
@@ -107,11 +107,14 @@ if(ENABLE_ELEMENTAL AND NOT ELEMENTAL_TAG)
     set(ELEMENTAL_TAG ${TA_TRACKED_ELEMENTAL_TAG})
 endif(ENABLE_ELEMENTAL AND NOT ELEMENTAL_TAG)
 
+option(ENABLE_SCALAPACK "Enable ScaLAPACK Bindings in TiledArray" OFF)
```

Comment:
should we just enable this by default? On x86 most should (will?) have MKL ... if enabled, and not found, it fails to build or ScaLAPACK just disabled?

Reply:
On systems that have MKL installed, I'm all for enabling by default, but we should have some logic to build ScaLAPACK if necessary if that's going to be the expected behaviour (for e.g. x64 AMD chips).

I think the trick here is asking will _**everyone**_ (or mostly everyone at least) that uses TA want the ScaLAPACK bindings? Or does that put an unnecessary dependency requirement for folks who just want to use the tensor contractions?

Reply:
The convention I have always followed is for `ENABLE_X=ON` to permit use of `X` if available, not declare that `X` is required. So `ENABLE_SCALAPACK=ON` would enable looking for it, but would not fail if not found. Building Scalapack (or BLAS/LAPACK) for that matter is a step too far right now (or for the foreseeable future).